### DISCUSSION TITLE
Are you monitoring your fiat exchange rate?

### DISCUSSION THREAD

**Discussion by markettes** [ACINQ/phoenix#366] [Q&A] [answered]
I was having some big difference(~10%) between Coingecko and your exchange rate for CZK here in BTC Prague.

**Comment by dpad85** [comment, marked as answer]
Hey, yes we are aware of that discrepancy in the Android app. It's an issue with the exchange rate provider we use on Android. It will be fixed in the next version (which is a major rework of the app).

**Reply by markettes** [reply]
Okay, thanks so much!
